source/_integrations/analytics.markdown

```diff
@@ -0,0 +1,143 @@
+---
+title: Analytics
+description: Share system analytics and diagnostics
+ha_category:
+  - Other
+ha_release: 2021.4 
+ha_iot_class: Cloud Push
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+ha_domain: analytics
+---
+
+The Analytics integration will collect information about the running Home Assistant instance and its environment. The information sent depends on what you enable, by default nothing is enabled. The different sections can be controlled in the UI under **{% my general title="Configuration >> General" %}**.
+
+**{% my general badge %}**
+
+When enabled the integration will send data 15 minutes after each start, and every 24h after startup. When the data is sent the full payload will be printed to your log.
+
+The collected data is available to the public at https://analytics.home-assistant.io
+
+## Configuration
+
+This integration is by default enabled, unless you've disabled or removed the [`default_config:`](/integrations/default_config/) line from your configuration. If that is the case, the following example shows you how to enable this integration manually:
```

Comment:
`This integration is by default enabled` could be confusing after the introduction paragraph says `The information sent depends on what you enable, by default nothing is enabled.` - May want to clarify that the integration is enabled by default but will not transmit anything by default.

Reply:
adjusted both sections